proposed/0007-Address_relocation.md

```diff
@@ -0,0 +1,40 @@
+- Feature Name: Prevent any single address being used more than once during address relocation.
+- Type enhancement
+- Related components routing
+- Start Date: 21-09-2015
+- RFC PR: (leave this empty)
+- Issue number: (leave this empty)
+
+# Summary
+
+Prevent a potential attack where a node gains overall/majority influence in a group setting.
```

Comment:
`node` here may give a wrong impression that the network allows a node having more weight.
I think it is more accurate to say: ... attack where a hacker spawn clustered nodes closing to each other to gain overall/majority ...